cli/ast_test_generator.php

```diff
@@ -306,13 +314,13 @@ function escp(string $string): string {
 
         $item = $indent2 . '$this->expect(' . escp($args[0]) .',' . $nl;
         $item .= $indent2 . '              ' . escp($args[1]) . ',' . $nl;
-        $item .= $indent2 . '              array(';
+        $item .= $indent2 . '              [';
```

Comment:
Maybe worth double checking this one as it's modifying a variable string value

Reply:
I updated this file manually in response to Matti's comment above to avoid us regenerating code with the old syntax. It's worth you going through this file carefully to make sure I've done it right.

Reply:
Just had a look over this file, looks good to me

Reply:
Thank you. I ran the script from the command line and the generated output matched the existing updated files.